scripts/helper/create-cluster.sh

```diff
@@ -276,48 +283,20 @@ case "$OCP_VERSION" in
 4.4|4.5)
 	patch -p1 < $WORKSPACE/ocs-upi-kvm/files/ocp4-upi-kvm.legacy.patch
 	;;
-*)
-	patch -p1 < $WORKSPACE/ocs-upi-kvm/files/ocp4-upi-kvm.patch
```

Comment:
Let's make this conditional on the presence of the patch also.   We may need to develop and test fixes to this release.